<?xml version="1.0" encoding="utf-8"?>

<launch>
	<arg name="marker_size" default="15" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />
	<arg name="cam_image_topic" default="/camera/image_raw" />
	<arg name="cam_info_topic" default="/camera/camera_info" />	
	<arg name="output_frame" default="/camera" />
	<arg name="max_frequency" default="10" /> 

	<arg name="broadcast_tf" default="false" /> <!-- tf cam -> marker (yes or no) -->

	<node name="ar_track_alvar" pkg="ar_track_alvar_fh" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame) $(arg max_frequency)">

	<param name="broadcast_tf" value="$(arg broadcast_tf)"/>

	</node>
</launch>
